nnfabrik/main.py

```diff
@@ -26,11 +26,62 @@ class Fabrikant(dj.Manual):
     definition = """
     fabrikant_name: varchar(32)       # Name of the contributor that added this entry
     ---
-    email: varchar(64)      # e-mail address
-    affiliation: varchar(32) # conributor's affiliation
-    dj_username: varchar(64) # DataJoint username
+    full_name="": varchar(128) # full name of the person
```

Comment:
Would this require us to drop the existing shared fabrikant table?

Reply:
No - this is only applicable to the new one you create.